### DISCUSSION TITLE
Elección del dataset de la entrega #1 del proyecto

### DISCUSSION THREAD

**Discussion by spmian** [rramosp/ai4eng.v1#241] [UDEA Students]
Profesor, buenos días! 
Me podría asesorar en la elección de una de las siguientes competencias que encontré, le anexo los enlaces. 
1. Reserva confirmada: https://www.kaggle.com/competitions/reserva-confirmada/overview
2. Porland Housing Age Prediction: https://www.kaggle.com/competitions/portland-housing-age-prediction-nwvs-s00e01/overview
3. Water Quality Prediction: https://www.kaggle.com/competitions/water-quality-prediction-rk/data

Los 3 me parecieron interesantes, pero no sé cual me recomienda usted, espero su respuesta.

**Comment by rramosp** [comment]
hazme un resumen MUY breve de cada uno por favor ... qué se quiere
predecir, cuantas filas, columnas y ficheros tienen los datos

On Mon, Mar 6, 2023 at 10:35 AM Miguel Sánchez ***@***.***>
wrote:

> Profesor, buenos días!
> Me podría asesorar en la elección de una de las siguientes competencias
> que encontré, le anexo los enlaces.
>
>    1. Reserva confirmada:
>    https://www.kaggle.com/competitions/reserva-confirmada/overview
>    2. Porland Housing Age Prediction:
>    https://www.kaggle.com/competitions/portland-housing-age-prediction-nwvs-s00e01/overview
>    3. Water Quality Prediction:
>    https://www.kaggle.com/competitions/water-quality-prediction-rk/data
>
> Los 3 me parecieron interesantes, pero no sé cual me recomienda usted,
> espero su respuesta.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/241>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKHCWG6GKKRO2C4DSLW2X72JANCNFSM6AAAAAAVRI57MQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by spmian** [reply]
1. Reserva confirmada: https://www.kaggle.com/competitions/reserva-confirmada/overview
- Que se busca predecir:  Si una reservacion será cancelada o no.
- El test.csv tiene 10884 filas y 18 columnas y el train.csv tiene 25393 filas y 19 columnas. 
- Tanto el test.csv como el train.csv tiene datos 
- El test.csv: es el conjunto de datos que debe predecirse.
- El train.csv: están los datos completos de reservas, indicando si fueron o no canceladas.

2. Porland Housing Age Prediction: https://www.kaggle.com/competitions/portland-housing-age-prediction-nwvs-s00e01/overview
- Se busca predecir el año de construcción de las casas en función de sus características. 
- El test.csv tiene 4197 filas y 15 columnas y el train.csv tiene 20001 filas y 16 columnas
- Los ficheros tiene datos tales como el houseID, yearBuilt, lastSoldPrice, entre otros. 

3. Water Quality Prediction: https://www.kaggle.com/competitions/water-quality-prediction-rk/data
-Se busca predecir la calidad del agua  basado en los datos de calidad del agua que se le proporcionaron en el archivo de entrenamiento y predecir aún más la estimación de calidad.
- El test.csv tiene 7001 filas y 27 columnas y el train.csv tiene 13001 filas y 28 columnas
- Los ficheros tiene datos tales como categorias, caracteristicas, resultados, entre otras

**Comment by rramosp** [comment]
hola miguel, ninguno de los tres cumple los requisitos de columnas. si en
el primero creas creas columnas nuevas para llegar a las requeridas podría
ser válido. tienes que evaluar si eso sería posible.

On Mon, Mar 6, 2023 at 10:57 PM Miguel Sánchez ***@***.***>
wrote:

>
>    1. Reserva confirmada:
>    https://www.kaggle.com/competitions/reserva-confirmada/overview
>
>
>    - Que se busca predecir: Si una reservacion será cancelada o no.
>    - El test.csv tiene 10884 filas y 18 columnas y el train.csv tiene
>    25393 filas y 19 columnas.
>    - Tanto el test.csv como el train.csv tiene datos
>    - El test.csv: es el conjunto de datos que debe predecirse.
>    - El train.csv: están los datos completos de reservas, indicando si
>    fueron o no canceladas.
>
>
>    1. Porland Housing Age Prediction:
>    https://www.kaggle.com/competitions/portland-housing-age-prediction-nwvs-s00e01/overview
>
>
>    - Se busca predecir el año de construcción de las casas en función de
>    sus características.
>    - El test.csv tiene 4197 filas y 15 columnas y el train.csv tiene
>    20001 filas y 16 columnas
>    - Los ficheros tiene datos tales como el houseID, yearBuilt,
>    lastSoldPrice, entre otros.
>
>
>    1. Water Quality Prediction:
>    https://www.kaggle.com/competitions/water-quality-prediction-rk/data
>    -Se busca predecir la calidad del agua basado en los datos de calidad
>    del agua que se le proporcionaron en el archivo de entrenamiento y predecir
>    aún más la estimación de calidad.
>
>
>    - El test.csv tiene 7001 filas y 27 columnas y el train.csv tiene
>    13001 filas y 28 columnas
>    - Los ficheros tiene datos tales como categorias, caracteristicas,
>    resultados, entre otras
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/241#discussioncomment-5223144>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZINQ6DJVG7RMHKRKND3W22W2NANCNFSM6AAAAAAVRI57MQ>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by spmian** [reply]
Profesor buenos días! 
y que tal estos dataset? 

1. Precio de alojamientos en Paises Bajo: https://www.kaggle.com/competitions/fcg-2022-netherlands-accommodation-prices/data?select=train.csv
- Objetivo: Predecir el precio de alquiler de venta para cada alojamiento.
- N° Filas y columnas (train.csv): 27899 filas y 33 columnas
- Ficheros: Tiene datos, tales como: Ciudad, latitud, longitud, área, amoblado, disponibilidad de internet....

2. Peso al nacer: https://www.kaggle.com/competitions/fall2022-birth-weight-prediction/data?select=data-description.txt
- Objetivo: predecir el peso de los bebes antes de nacer. 
- N° Filas y columnas: 101401 filas y 37 columnas (bebé-pesos-dataset.csv) - 2000 filas y 36 columnas(judge-without-labels.csv)
- Ficheros: Tiene datos tales como: Sex, marital, age the father, Number of prenatal visits,  Number of other terminations....

3. Cliente en riesgo de incumplir préstamo: https://www.kaggle.com/competitions/ai511-homeloan-2022/data?select=train_data.csv
- Objetivo: Predecir si un cliente está en riesgo de incumplir su préstamo
-  N° Filas y columnas: 184507 filas y 121 columnas (train.csv)
- Ficheros: Tienen datos tales como: si el cliente tiene coche, números de hijos del cliente, ingresos del cliente....

**¿Cuál me recomienda?**

**Comment by rramosp** [comment]
hola miguel, cualquiera de los tres es adecuado. me parecen màs interesante
sel primero y el ùltimo, ya que me parece que tienen màs variedad de
informaciòn con la que jugar y aprender

On Wed, Mar 8, 2023 at 9:58 AM Miguel Sánchez ***@***.***>
wrote:

> Profesor buenos días!
> y que tal estos dataset?
>
>    1. Precio de alojamientos en Paises Bajo:
>    https://www.kaggle.com/competitions/fcg-2022-netherlands-accommodation-prices/data?select=train.csv
>
>
>    - Objetivo: Predecir el precio de alquiler de venta para cada
>    alojamiento.
>    - N° Filas y columnas (train.csv): 27899 filas y 33 columnas
>    - Ficheros: Tiene datos, tales como: Ciudad, latitud, longitud, área,
>    amoblado, disponibilidad de internet....
>
>
>    1. Peso al nacer:
>    https://www.kaggle.com/competitions/fall2022-birth-weight-prediction/data?select=data-description.txt
>
>
>    - Objetivo: predecir el peso de los bebes antes de nacer.
>    - N° Filas y columnas: 101401 filas y 37 columnas
>    (bebé-pesos-dataset.csv) - 2000 filas y 36
>    columnas(judge-without-labels.csv)
>    - Ficheros: Tiene datos tales como: Sex, marital, age the father,
>    Number of prenatal visits, Number of other terminations....
>
>
>    1. Cliente en riesgo de incumplir préstamo:
>    https://www.kaggle.com/competitions/ai511-homeloan-2022/data?select=train_data.csv
>
>
>    - Objetivo: Predecir si un cliente está en riesgo de incumplir su
>    préstamo
>    - N° Filas y columnas: 184507 filas y 121 columnas (train.csv)
>    - Ficheros: Tienen datos tales como: si el cliente tiene coche,
>    números de hijos del cliente, ingresos del cliente....
>
> *¿Cuál me recomienda?*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/241#discussioncomment-5242748>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKZ4CHCVRUFOBC6FV3W3CNCNANCNFSM6AAAAAAVRI57MQ>
> .
> You are receiving this because you commented.Message ID: <rramosp/ai4eng.
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by spmian** [reply]
OK, muchas gracias profesor
